examples/CSVImporter/CSVImporter.go

```diff
@@ -137,25 +134,38 @@ func ProcessCSVRecord(jc jcapi.JCAPI, userList []jcapi.JCUser, csvRecord []strin
 		opCode = jcapi.Insert
 	}
 
+	if currentUser.Uid != "" || currentUser.Gid != "" {
+		currentUser.EnableManagedUid = true
+	}
+
 	// Perform the requested operation on the current user and report results
-	currentUserId, err := jc.AddUpdateUser(opCode, currentUser)
+	currentUserId, err = jc.AddUpdateUser(opCode, currentUser)
 
 	if err != nil {
-		fmt.Printf("Could not process user '%s', err='%s'", currentUser.ToString(), err)
+		err = fmt.Errorf("Could not %s user '%s', err='%s'", jcapi.MapJCOpToHTTP(opCode), currentUser.ToString(), err)
 		return
 	} else {
-		fmt.Printf("Processed user ID '%s'\n", currentUserId)
+		if opCode == jcapi.Update {
+			fmt.Printf("\tUser '%s' (ID '%s') updated from input file\n", currentUser.UserName, currentUserId)
+		} else {
+			fmt.Printf("\tLoaded user '%s' (ID '%s')\n", currentUser.UserName, currentUserId)
+		}
 	}
 
 	// Create/associate JumpCloud tags for the host and user...
 	if currentHost != "" {
```

Comment:
changing this to 
if currentHost == ""{
  return;
}
would eliminate 75 lines of indentation